### PR TITLE
Isis fund

### PR DESCRIPTION
**Pull request by jrcain-usgs**
## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: docs/how-to-guides/software-management/software-support.md
Comment (amystamile-usgs) [inline docs/how-to-guides/software-management/software-support.md:1]: @jrcain-usgs Why are there software-management updates in this PR?

Comment (amystamile-usgs) [reply docs/how-to-guides/software-management/software-support.md:1]: I rebased my branch and it fixed the issue.

**Comment by amystamile-usgs** [comment]
@jrcain-usgs This branch just needs a rebase against my branch. 
`git fetch <amy-branch>` 
`git rebase <amy-branch>/isis-fundamentals`

or if it's easier, pull a new branch from my branch and cherry-pick your changes.

**Comment by jrcain-usgs** [comment]
@amystamile-usgs rebased, should be good to go now

**Pull request merged**